contracts/SortitionTree.sol

```diff
@@ -35,7 +35,14 @@ contract SortitionTree {
   // level6 256k
   // level7 2M
   uint256 internal root;
+
+  // A mapping from layer => (index => branch). For example, to access the 6th
+  // (0-index) branch in the 2nd layer (right below the root node), call
+  // branches[2][6]. Mappings are used in place of arrays for efficiency.
```

Comment:
If we want to access the 6th branch and branches are 0-index, shouldn't we call `branches[2][5]`? Also, worth dropping a note about layer indexing because it's not clear whether they are 0-index or not.